test/clear_module/src/index.md

```diff
@@ -21,3 +21,33 @@ a = MyMutableStruct(1)
 Once Documenter is finished processing this page, it should remove the
 reference to the object `a` and a subsequent garbage collection should free
 it, which we can later test by observing the value of `Main.finalizer_count`.
+
+# Example blocks
+
+```@example example-1
+a = 1 # simple assignment
+```
+
+```@example example-2
+nothing = 1 # it is possible to use `nothing` as a variable name
+a = 2
+```
+
+```@example example-3
+nothing() = 3 # it is possible to use `nothing` as a function name
+a = nothing()
+```
+
+```@example example-4
+@static if VERSION >= v"1.8"
+typed::String = "string" # it is possible to use `::` to specify the type of a variable
```

Comment:
Can we detect this (and how) to properly fix the issue (in addition to also having the try/catch)